### PR TITLE
Domain Analysis documents

### PR DESCRIPTION
**Pull request by megascrapper**
All the files that are submitted for domain analysis assignment, except the work breakdown structure. This includes:
- Exported PDF of the report
- Conceptual class diagram
- Licence file

### CODE REVIEW

**Review by ghost — COMMENTED** [review]
Is there any reason for PR'ing now, rather than waiting for the WBS to be completed as well?

**Comment by megascrapper** [comment]
Not yet, just review it whenever all files are ready.
I was meant to create a draft PR but for some reason I can't find the button to do it.

**Comment by megascrapper** [comment]
@Brittank88 feel free to merge this branch as we've finished the domain analysis.

File: domain_analysis/class_diagram.uxf [hunk 1]
```diff
@@ -0,0 +1,297 @@
...
+  <element>
+    <id>Relation</id>
+    <coordinates>
+      <x>380</x>
+      <y>370</y>
+      <w>140</w>
+      <h>120</h>
+    </coordinates>
+    <panel_attributes>lt=&lt;.
+porocessed into v</panel_attributes>
```
Comment (ghost) [inline domain_analysis/class_diagram.uxf]:
```suggestion
processed into v</panel_attributes>
```

**Review by ghost — APPROVED** [review]

**Pull request merged**